### PR TITLE
Make AsyncActor#wait_until_async_stopped method public

### PR DESCRIPTION
**Pull request by hxiong388**

### CODE REVIEW

File: stackdriver-core/lib/stackdriver/core/async_actor.rb [hunk 1]
```diff
@@ -199,6 +199,30 @@ def async_stop!
       end
 
       ##
+      # Block current thread until the async job is fully stopped.
```
Comment (blowmage) [inline stackdriver-core/lib/stackdriver/core/async_actor.rb]: Shouldn't this still be labeled `@private`? You want to be able to call this method from other objects as part of the implementation, but you don't want this method to be considered part of the public API, right?

Comment (hxiong388) [inline stackdriver-core/lib/stackdriver/core/async_actor.rb]: @dazuma is arguing we should make this method public public, in the same way that `AsyncWriter#wait_until_stopped` was public to begin with.

Comment (dazuma) [inline stackdriver-core/lib/stackdriver/core/async_actor.rb]: And I think the method is useful to clients and should be part of the interface.

Comment (blowmage) [inline stackdriver-core/lib/stackdriver/core/async_actor.rb]: kk

**Pull request merged**